pandas/tests/tslibs/test_timedeltas.py

```diff
@@ -34,3 +34,41 @@ def test_huge_nanoseconds_overflow():
     # GH 32402
     assert delta_to_nanoseconds(Timedelta(1e10)) == 1e10
     assert delta_to_nanoseconds(Timedelta(nanoseconds=1e10)) == 1e10
+
+
+class TestComputingDeltasWithTimezoneDifferences:
+    def test_fails_to_subtract_when_one_date_is_naive(self):
```

Comment:
these should go in tests.scalar.timestamp.test_arithmetic

Reply:
ok, it seems like there are also relevant tests in `tests/arithmatic/test_timedelta64.py`. i see a note in that file about needing to do deduplication and parametrization of the tests (oo, and it looks like you might have been the author of that note haha)